OPERATOR.rst

```diff
@@ -432,6 +432,31 @@ SSH into the instance, and run ``sudo yum update`` followed by ``sudo reboot``.
 Wait for the GitLab web application to become available again and perform a
 ``git fetch`` from one of the Git repositories hosted on that instance.
 
+Export AWS Inspector findings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. ``_select anvilprod``
+
+#. Run ``python scripts/export_inspector_findings.py`` to generate a CSV file
+
+#. Open the `Anvilprod Inspector Findings spreadsheet`_
+
+#. Import the CSV file with the following import options:
+
+    - Import location: Insert new sheet(s)
+
+    - Convert text to numbers, dates, and formulas: Checked
+
+#. Rename the new tab using `YYYY-MM-DD` format and move it to the front of the
```

Comment:
Note that the last generated sheet is also post-fixed with `in-boundary`. Maybe confirm the naming convention.